.github/workflows/multichain-integration.yml

```diff
@@ -43,7 +44,7 @@ jobs:
       - name: Pull Relayer & Sandbox Docker Images
         run: |
           docker pull ghcr.io/near/os-relayer:12ba6e35690df3979fce0b36a41d0ca0db9c0ab4
-          docker pull ghcr.io/near/near-lake-indexer:e6519c922435f3d18b5f2ddac5d1ec171ef4dd6b
+          docker pull ghcr.io/near/near-lake-indexer:node-1.38
```

Comment:
why node-1.38?

Reply:
because node-1.39 doesn't work well with `near-crypto 0.21`. I'll update to `near-crypto 0.22` afterwards which does. Just didn't want to add more complexity to this PR